lib/srv/sess.go

```diff
@@ -1462,11 +1518,19 @@ func (s *session) checkPresence() error {
 	return nil
 }
 
+// fileTransferRequest is a request to upload or download a file from the node.
 type fileTransferRequest struct {
+	id string
 	// requester is the Teleport User that requested the file transfer
 	requester string
-	// shellCmd is the requested scp command to run
-	shellCmd string
+	// direction is "upload" or "download"
+	direction string
```

Comment:
Nit: could this be a `bool` or typed constant instead as the number of valid values here is very limited (2)?

Reply:
I'll change this to a bool but also change the name to `Download`, that way its `Download = true` rather than `Direction = true` (doesn't make much sense).